ephios/core/signals.py

```diff
@@ -101,6 +101,15 @@
 Receivers should return html that is added below the participations.
 """
 
+shift_action = PluginSignal()
+"""
+This signal is sent out to collect additional actions that managers can perform on on a shift. For
+each action, a button will be displayed in the shift card next to the disposition button. Receivers
+of the signal will receive the ``shift`` and ``request`` and are expected to return an array of
+``{label: str, url: str}`` dicts, representing the available actions.
+The buttons will only be shown to managers of the respective shift.
+"""
+
```

Comment:
We should be careful with the naming here. "event_action" refers to the dropdown menu on event, so I'd expect "shift_action" to also be about the dropdown menu. I think renaming `event_action` to `event_menu` is the better way out though.

Reply:
Naming-wise, I agree that I prefer renaming the event-related signal.

However, this would be a breaking change. Is that okay?

Reply:
We will have breaking signal renaming in the next release any way (due to #1606), so lets do it now. There aren't many plugins we have that use the signal. Other people will get the error. This is why we are still not at version 1. ^^